features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/Comment.jsx

```diff
@@ -327,11 +334,56 @@ class Comment extends React.Component {
                                             allComments={allComments}
                                         />
                                     )} */}
+                                        {comment.replies && comment.replies.list.slice(0).map((replie, index) => (
```

Comment:
shall we change variable name **replie** to **reply**

Reply:
Fixed